wsnet/wsnet_test.go

```diff
@@ -68,7 +68,7 @@ func createDumbBroker(t testing.TB) (connectAddr string, listenAddr string) {
 		mut.Lock()
 		defer mut.Unlock()
 		if sess == nil {
-			t.Error("listen not called")
+			_, _ = io.Copy(io.Discard, nc)
```

Comment:
??

Reply:
I'll add a comment. This is to emulate a PubSub style scenario where we don't know if anyone's listening.